system-parachains/encointer/src/lib.rs

```diff
@@ -122,7 +125,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("encointer-parachain"),
 	impl_name: create_runtime_str!("encointer-parachain"),
 	authoring_version: 1,
-	spec_version: 1_002_000,
+	spec_version: 1_002_010,
```

Comment:
Why this bump? I think we just bump everything to `1_003_000` on the next release, unless you want to make an intermediate release?

Reply:
Yeah, I can revert that